cni-plugin/pkg/install/install_test.go

```diff
@@ -98,6 +108,54 @@ func runCniContainer(tempDir string, binFolderWriteable bool, extraArgs ...strin
 	return err
 }
 
+func createKubernetesClient() *kubernetes.Clientset {
+	certsPath := os.Getenv("CERTS_PATH")
```

Comment:
This is a bit more involved than we usually need for loading kubeconfig. 

I think we'll want something like this: https://github.com/projectcalico/calico/blob/9687764b6e9a8ebc0592e3c811b4d8a100fd7b6a/kube-controllers/cmd/kube-controllers/main.go#L322-L327

```
clientcmd.BuildConfigFromFlags
```

Takes an optional kubeconfig file path, and loads and uses it if given. If not provided, it falls back on the InClusterClient. The kubeconfig file should include references to the necessary certificates, etc, so no need to explicitly load them here.

This is more consistent with how we configure kubeconfig files elsewhere, respecting the KUBECONFIG env var instead of a directory in which to search.

Reply:
@caseydavenport In `hack/test/certs/kubeconfig` the certificate files are hard-coded to `/home/user/certs/...`. While this works nicely with containers as you can mount them into the container where you like, it does not fit to the cni-plugin test driver, which runs outside of a container as far as I understand. That is the sole reason why the certificates were loaded manually.
Do you suggest to create a separate kubeconfig file or did you have something else in mind?